Jint/Native/Promise/PromiseState.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
```

Comment:
nit: If you have analysis tool like R# or Rider, there are some files where there are usings like these and they are not needed.

Reply:
Fixed